backend/resources/transactions.py

```diff
@@ -66,4 +66,5 @@ def get(self, email):
             if g.user != user:
                 return {"message": "Invalid user, can only be yourself"}, 401
             transactions = TransactionsModel.query.filter_by(user_id=user.id).all()
-            return {'transactions': [transaction.json() for transaction in user.transactions]}, 200
+            grouped_transactions = [[t.json() for t in transactions if t.id_transaction == i] for i in set([t.id_transaction for t in transactions])]
```

Comment:
Possible solucio al comentari del bot:
```suggestion
            grouped_transactions = [[t.json() for t in transactions if t.id_transaction == i] for i in set(t.id_transaction for t in transactions)]
```
Igualment aquesta funcio te cost O(n^2) es podria ordenar la llista per reduir el cost a nomes O(n^log n), utilitzant `group_by` de `itertools` o un [`defaultdict`](https://stackoverflow.com/questions/6602172/how-to-group-a-list-of-tuples-objects-by-similar-index-attribute-in-python). Una altre possible solucio seria utilizar el metode de `grup_by` en la query.